qcodes/utils/validators.py

```diff
@@ -301,6 +303,28 @@ def validate(self, value: numbertypes, context: str = '') -> None:
         super().validate(castvalue, context=context)
 
 
+class ComplexNum(Validator):
```

Comment:
Should this be renames ComplexNumbers for consistency

Reply:
yes :)